content/docs/reference/media/external/s3.mdx

```diff
@@ -90,6 +90,37 @@ i.e. You can disable `block public access settings` and set up the bucket policy
 }
 ```
 
+### 4. You should ensure the CORS configuration is set up correctly.
+
+Images are uploaded using S3 pre-signed URLs. This requires the bucket to have a CORS configuration that allows the origin of the request to access the bucket.
+
+```json
+[
+    {
+        "AllowedHeaders": [
+            "Authorization",
+            "Content-Range",
+            "Accept",
+            "Content-Type",
+            "Origin",
+            "Range"
+        ],
+        "AllowedMethods": [
+            "PUT"
+        ],
+        "AllowedOrigins": [
+            "*"
+        ],
+        "ExposeHeaders": [
+            "Content-Range",
+            "Content-Length",
+            "ETag"
+        ],
+        "MaxAgeSeconds": 3000
+    }
+]
+```
```

Comment:
**suggestion**: The JSON example above in step 3 uses 2-space indent rather than 4-space - I suggest updating this example to match.

```suggestion
```json
[
  {
    "AllowedHeaders": [
      "Authorization",
      "Content-Range",
      "Accept",
      "Content-Type",
      "Origin",
      "Range"
    ],
    "AllowedMethods": [
      "PUT"
    ],
    "AllowedOrigins": [
      "*"
    ],
    "ExposeHeaders": [
      "Content-Range",
      "Content-Length",
      "ETag"
    ],
    "MaxAgeSeconds": 3000
  }
]
```
```